src/unicorn_baseline/language/main.py

```diff
@@ -12,6 +12,200 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+"""
+The following is a simple example algorithm.
+
+It is meant to run within a container.
+
+To run it locally, you can call the following bash script:
+
+  ./do_test_run.sh
+
+This will start the inference and reads from ./test/input and writes to ./test/output
+
+To save the container and prep it for upload to Grand-Challenge.org you can call:
+
+  ./do_save.sh
+
+Any container that shows the same behaviour will do, this is purely an example of how one COULD do it.
+
+Reference the documentation to get details on the runtime environment on the platform:
+https://grand-challenge.org/documentation/runtime-environment/
+
+Happy programming!
+"""
+
+import json
+import time
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+from dragon_baseline.main import DragonBaseline
+from llm_extractinator import extractinate
+
+INPUT_PATH = Path("/input")
```

Comment:
These parameters should not be redefined here, but rather use the information propagated from the main inference script.